store/tikv/coprocessor.go

```diff
@@ -515,13 +515,24 @@ const minLogCopTaskTime = 300 * time.Millisecond
 // run is a worker function that get a copTask from channel, handle it and
 // send the result back.
 func (worker *copIteratorWorker) run(ctx context.Context) {
-	defer worker.wg.Done()
+	defer func() {
+		worker.actionOnExceed.broadcastExceed()
```

Comment:
why not close?

Reply:
If the concurrency is much bigger than count of the tasks like (30 workers, 10 tasks), the redundant workers would directly close the action.

Reply:
updated.